### ISSUE TITLE
Make Copilot Chat View searchable

### ISSUE THREAD

**Issue by Maddimax** [microsoft/vscode-copilot-release#124]
Add a "find" function, same as in the text editor, so the user can search for text in the chat window

**Comment by Daryubel** [comment]
Agree, currently you can only scroll through the chat window and try to find things, but if you're asking it to generate codes the response would be very long and hard to locate certain texts.

**Comment by jbhanks** [comment]
Has there been any progress? My chats get very long sometimes and if Copilot takes me down a detour rabbit-hole, it would be nice for it to be easier to find stuff earlier in the history. This is an extremely important feature to have.

**Comment by MosheTzvi** [comment]
ridiculous that this doesn't exist.  Especially given how obnoxiously verbose the copilot chat is.  It's incredibly important to be able to go back and find a previous message.

**Comment by yCodeTech** [comment]
Yeah, why isn't this added yet?
